reg/correctMotion_DFT.m

```diff
@@ -0,0 +1,64 @@
+function correctMotion_DFT(hdf5_dir,hdf5_in_name,hdf5_out_name)
+
+dataset = '/Data/Images';
+
+h5att = h5info(fullfile(hdf5_dir,hdf5_in_name),dataset);
+imageStackSize = h5att.Dataspace.Size;
+rows = imageStackSize(1);
+cols = imageStackSize(2);
+frames = imageStackSize(3);
+baseImage = h5read(fullfile(hdf5_dir,hdf5_in_name),dataset,[1,1,1],[rows,cols,1]);
+
+ssm_radius = 20;
+asm_radius = 5;
+
+hDisk  = fspecial('disk', ssm_radius);
+hDisk2 = fspecial('disk', asm_radius);
+transform = @(A) mosaic_transform(A, hDisk, hDisk2);
+
+im_ref = transform(baseImage);
+
+% Specify ROI
+%------------------------------------------------------------
+h_roi = figure;
+imagesc(im_ref); axis image; colormap gray;
+title('Select ROI');
+h_rect = imrect;
+mask_rect = round(getPosition(h_rect));
+cropBase = im_ref(mask_rect(2):mask_rect(2)+mask_rect(4),mask_rect(1):mask_rect(1)+mask_rect(3));
```

Comment:
Minor comment. Given that you use `mask_rect(2):...` here and also in the subsequent loop (Line 41), you should make the indices into its own variable.